yb-voyager/cmd/importData.go

```diff
@@ -397,6 +393,10 @@ func importData(importFileTasks []*ImportFileTask) {
 		utils.ErrExit("failed to get migration UUID: %w", err)
 	}
 
+	if (importerRole == TARGET_DB_IMPORTER_ROLE || importerRole == IMPORT_FILE_ROLE) && (tconf.EnableUpsert) {
+		utils.PrintAndLog(color.RedString("WARNING: Ensure that tables on target YugabyteDB do not have secondary indexes. If a table has secondary indexes, setting --enable-upsert to true may lead to corruption of the indexes."))
```

Comment:
If user has already enabled upsert, but based on this warning they want to disable it.

Shall we have it as a prompt so that user can exit and restart if required?

Reply:
Good point, let me do this.